Source/santactl/Commands/SNTCommandMetricsTest.m

```diff
@@ -32,12 +36,24 @@ - (void)setUp {
     exit(1);
   }
 
-  self.tempDir =
-    [[NSFileManager defaultManager] stringWithFileSystemRepresentation:tempPath
-                                                                length:strlen(tempPath)];
+  self.tempDir = [[NSFileManager defaultManager]
+      stringWithFileSystemRepresentation:tempPath
+                                  length:strlen(tempPath)];
+  // mock the SNTConfigurator
+  self.mockConfigurator = OCMClassMock([SNTConfigurator class]);
+  OCMStub([self.mockConfigurator configurator])
+      .andReturn(self.mockConfigurator);
+  OCMStub([self.mockConfigurator exportMetrics]).andReturn(YES);
+  OCMStub([self.mockConfigurator metricFormat])
+      .andReturn(SNTMetricFormatTypeMonarchJSON);
+  OCMStub([self.mockConfigurator metricURL])
+      .andReturn([NSURL URLWithString:@"http://localhost:2444/submit"]);
+  OCMStub([self.mockConfigurator metricExportInterval])
+      .andReturn((NSUInteger)30);
 }
 
 - (void)tearDown {
+  [self.mockConfigurator stopMocking];
```

Comment:
You don't need to (and shouldn't) call stopMocking unless you're doing so mid-test.

Reply:
Yup. Removed.